app/enterprise/2.3.x/kong-manager/service-directory-mapping.md

```diff
@@ -62,6 +62,8 @@ admin_gui_session_conf = { "secret":"set-your-string-here" }
 
 Configure LDAP Authentication for Kong Manager with the following properties. Note the attribute variables are defined below:
 
+For Active Directory environments, instead of port 389, user the Global Catalog port which is 3268 by default.
```

Comment:
This might make more sense as part of the breakdown of properties below the codeblock, right after `ldap_password`. Something like this:

* `ldap_port`: the LDAP port, which defaults to `389`. 
  * For Active Directory environments, use the Global Catalog port, which is `3268` by default.

Reply:
"defaults" is not a word but is frequently in docs written by developers. I'll check out the branch and edit.